README.md

```diff
@@ -1,7 +1,6 @@
 
-[![Build Status](https://travis-ci.com/key4hep/EDM4HEP.svg?branch=master)](https://travis-ci.com/key4hep/EDM4HEP)
-[![linux](https://github.com/key4hep/EDM4hep/workflows/linux.yml/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/linux.yml)
-[![key4hep](https://github.com/key4hep/EDM4hep/workflows/key4hep.yml/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/key4hep.yml)
+[![linux](https://github.com/key4hep/EDM4hep/workflows/lcg_linux/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/lcg_linux.yml)
+[![key4hep](https://github.com/key4hep/EDM4hep/workflows/key4hep_linux/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/key4hep_linux.yml)
```

Comment:
```suggestion
[![lcg_linux](https://github.com/key4hep/EDM4hep/actions/workflows/lcg_linux.yml/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/lcg_linux.yml)
[![key4hep_linux](https://github.com/key4hep/EDM4hep/actions/workflows/key4hep_linux.yml/badge.svg)](https://github.com/key4hep/EDM4hep/actions/workflows/key4hep_linux.yml)
```

I missed the `actions` in the original urls. Whether we want badges to just read `linux` or `lcg_linux` doesn't really matter. I simply copied the badge urls from the actions directly, where the filenames are used. For the other packages we only use `linux`.

Reply:
Would have worked in my version too, if you remove the `.yml` in the url, you don't need the `/actions/`

Reply:
Ah, I didn't know that. TIL, thanks :)